CMSgov/cms-carts-seds
# [CMDCT-3817] Upgrade React-Router-Dom

### Description

The long awaited react-router upgrade is here, and the fruits of our labor can finally be fully observed. Bask in the hooks that make this a lot easier to use router!

### Related ticket(s)


CMDCT-3817

---
### How to test

Log in as a stateuser
Enter a report
Navigate through the sections, clicking any buttons you see to go to different pages. I.E., check the Sidebar, the print actions button on the bottom, the navigation buttons on the bottom, etc.
If everything loads, fill in some data and make sure it saves
Use the top right menu to check that your state user info shows up in the account page
Click the banner image to see you navigate to the root page
Sign out
Sign in as an admin
Repeat the same steps as above and it should all work


### Notes


React-Router is removed
React-Router-Dom 5.2 -> 6.26

---
### Pre-review checklist


---
### Pre-merge checklist


#### Review
